SFLoggingKit/Classes/SFLogger.swift

```diff
@@ -8,34 +8,302 @@
 
 import Foundation
 
-open class SFLogger: SFLoggingInterface {
-    
-    public static let shared = SFLogger()
-    
-    init() {
-        currentLogLevel = .debug
-        shouldLogInBackgroundConsole = false
-    }
-    
-    final public var currentLogLevel: SFLogLevel
-    final public var shouldLogInBackgroundConsole: Bool
-    
-    final public func configure(logLevel: SFLogLevel, shouldLogInBackgroundConsole: Bool) {
-        currentLogLevel = logLevel
-        self.shouldLogInBackgroundConsole = shouldLogInBackgroundConsole
-    }
-    
-    final public func log(_ message: String, logLevel: SFLogLevel) {
-        if logLevel.rawValue >= currentLogLevel.rawValue {
-            printMessage(message)
-        }
+// NOTE: Consider adding Thread name in log
+// NOTE: Consider adding column number in log
+// NOTE: Consider support for message id
+// NOTE: Consider adding filters
+
+// MARK: - Interface
+
+open class SFLogger {
+
+    /// Default instance. Writes to console using `print()`. Adds Timestamp, file name, calling function, line and log level name
+    public static let `default`: SFLogger = {
+        SFLogger(
+            minLevel: .debug,
+            writers: [SFConsoleLogWriter()],
+            modifiers: [
+                // NOTE: Will be printed in reverse order
+                SFLevelNameLogMessageModifier(),
+//                SFSymbolLogMessageModifier(),
```

Comment:
Why do we have commented out code here?

Reply:
As something that should be considered. If we decide we need log level symbol in the `default` log message we can remove the comment.